mapApp/static/mapApp/js/map.js

```diff
@@ -11,7 +11,7 @@ var MapQuestOpen_OSM = L.tileLayer('http://otile{s}.mqcdn.com/tiles/1.0.0/map/{z
       attribution: '<a href="https://openstreetmap.org/">&copy OpenStreetMap contributors, CC-BY-SA</a>',
       subdomains: '1234'
     }),
-    stravaHM = L.tileLayer('https://globalheat.strava.com/tiles/cycling/color5/{z}/{x}/{y}.png', {
+    stravaHM = L.tileLayer('https://heatmap-external-a.strava.com/tiles/ride/gray/{z}/{x}/{y}.png', {
```

Comment:
I'm pretty sure the Leaflet API supports multiple domain sources for tiles. You should be able to substitute the 'a' with {s}. The Leaflet API will then automatically delegate requests to subdomains a, b and c.

Reply:
Good idea - thanks!